auth-server/src/services/auth.ts

```diff
@@ -557,36 +558,17 @@ export default class AuthService {
 			return;
 		}
 
-		const expires = new Date(Date.now() + ONE_DAY).toISOString(); // 24 hours
+		const resetToken = uuid();
 
-		const resetToken = await PasswordResetToken
-			.query()
-			.allowInsert('[token, user_id, valid, expires]')
-			.insert({
-				token: uuid(),
-				user_id: user.id,
-				valid: true,
-				expires
-			});
+		await redisSetex(getPwdResetTokenKey(resetToken), ONE_DAY, `${user.id}`);
 
 		sendResetPasswordEmail(user, resetToken);
 	}
 
 	public async ResetPassword(token: string, newPassword: string) {
-		const resetToken = await PasswordResetToken
-			.query()
-			.where('token', token)
-			.first();
-
-		if (!resetToken) {
-			throw new AuthenticationError(messages.PASSWORD_RESET_TOKEN_NOT_FOUND);
```

Comment:
This message should be removed because it's not used any more, except in the test that are actually failing.